content/applications/general/voip/voip_widget.rst

```diff
@@ -0,0 +1,76 @@
+===========
+VoIP widget
+===========
+
+Phone calls
+===========
+
+Make phone calls by clicking on the :guilabel:`📞 (phone)` icon in the top navigation bar. A widget
+appears in the lower left corner of the Odoo dashboard. Freely navigate throughout the database
+while making and receiving calls from the widget.
+
+When receiving call in Odoo, the :abbr:`VoIP (Voice over Internet Protocol)` widget rings and
+displays a notification. To minimize the widget click the :guilabel:`X` icon in the upper right of
+the screen.
+
+.. note::
+   The :abbr:`VoIP (Voice over Internet Protocol)` number is the one provided by Axivox. It can be
+   accessed by navigating to `https://manage.axivox.com/ <https://manage.axivox.com/>`_. After
+   logging into the portal go to :menuselection:`Users --> Outgoing number` (column).
```

Comment:
```suggestion
To make phone calls while in the Odoo database, click the :guilabel:`📞 (phone)` icon, located in the
top navigation bar.

When clicked, a :guilabel:`VOIP` pop-up widget appears in the lower-left corner of the Odoo
database. The widget allows users to freely navigate throughout the database, while making and
receiving calls.

When receiving calls in Odoo, the :guilabel:`VOIP` widget rings,and displays a notification. To
close the widget, click the :guilabel:`X` icon in the upper-right of the widget's screen.

.. note::
   The :abbr:`VoIP (Voice over Internet Protocol)` number is the one provided by Axivox. It can be
   accessed by navigating to `https://manage.axivox.com/ <https://manage.axivox.com/>`_. After
   logging into the portal, go to :menuselection:`Users --> Outgoing number` (column).
```